src/Symfony/Component/Console/Application.php

```diff
@@ -594,6 +594,8 @@ public function all($namespace = null)
      */
     public static function getAbbreviations($names)
     {
+        @trigger_error(sprintf('%s is deprecated as of 3.2 and will be removed in 4.0.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
@geoffrey-brier in any case, this has nothing to do in a PR adding more code coverage